### PR TITLE
#1305 replace markup via link

### PR DESCRIPTION
**Pull request by Gormartsen**
Fixes https://github.com/backdrop/backdrop-issues/issues/1305

### CODE REVIEW

File: core/modules/views_ui/views_ui.admin.inc [hunk 1]
```diff
@@ -442,8 +442,17 @@ function views_ui_add_form($form, &$form_state) {
     '#validate' => array('views_ui_wizard_form_validate'),
     '#submit' => array('views_ui_add_form_save_submit'),
   );
+  
+  $path = 'admin/structure/views'
```
Comment (quicksketch) [inline core/modules/views_ui/views_ui.admin.inc]: This line is missing a semicolon, causing a parse error.

Comment (Gormartsen) [inline core/modules/views_ui/views_ui.admin.inc]: Ouch.
Will update soon. It's quite strange!

Regards,
Gor Martsen
@gormartsen

**Comment by Gormartsen** [comment]
@quicksketch 

Looks like core/modules/user/tests/user.test fail on next code:

```
    $status_result = $this->xpath('//div[contains(@class, "view-user-admin")]//table/tbody/tr[position()=4]//td[position()=3]');
    $this->assertEqual(trim($status_result[0]), 'Blocked', 'User C blocked');
```

```
    $this->assertMail("to", $account->mail, "Activation mail sent to user C");
```

**Comment by quicksketch** [comment]
Doh, this is a "random" failure. I'll restart the tests.

File: core/modules/views_ui/views_ui.admin.inc [hunk 2]
```diff
@@ -442,8 +442,17 @@ function views_ui_add_form($form, &$form_state) {
     '#validate' => array('views_ui_wizard_form_validate'),
     '#submit' => array('views_ui_add_form_save_submit'),
   );
+  
+  $path = 'admin/structure/views';
+  $options = backdrop_parse_url($path);
```
Comment (quicksketch) [inline core/modules/views_ui/views_ui.admin.inc:447]: Let's just remove this line. With a hard-coded path like 'admin/structure/views', it won't ever create any `$options` that we care about (e.g. a `query` value).

**Comment by Gormartsen** [comment]
Nice! All checks have passed.

**Comment by quicksketch** [comment]
Thanks @Gormartsen! I squashed the commits into a single one and committed it here: https://github.com/backdrop/backdrop/commit/192bcc2ea0328864f54bdba44ef0f9f37f7d5ec9.

**Pull request closed without merging**